flutter/shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -108,6 +108,8 @@ void PlatformViewChannel::HandleMethodCall(
     OnResize(arguments, std::move(result));
   } else if (method == "touch") {
     OnTouch(arguments, std::move(result));
+  } else if (method == "offset") {
+    OnOffset(arguments, std::move(result));
```

Comment:
I think it looks better to use them in the order they are defined.
"dispose"-> "offset"-> "resize"-> "touch"

Reply:
Modified. thank you.